src/AsyncWebSocket.h

```diff
@@ -255,6 +255,7 @@ class AsyncWebSocket: public AsyncWebHandler {
 
     void close(uint32_t id, uint16_t code=0, const char * message=NULL);
     void closeAll(uint16_t code=0, const char * message=NULL);
+    void cleanupClients(uint16_t maxClients = 4);
```

Comment:
this number should really equal the max open TCP connections of the device. for ESP8266 that is 4 or 5 and for ESP32 that is 16. Having it 4 by default will make it limited on ESP32